app/src/main/java/com/team7/cmput301/android/theirisproject/activity/BodyPhotoListActivity.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Team 7, CMPUT301, University of Alberta - All Rights Reserved. You may use, distribute, or modify this code under terms and conditions of the Code of Students Behavior at University of Alberta
+ */
+
+package com.team7.cmput301.android.theirisproject.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.widget.GridLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import com.team7.cmput301.android.theirisproject.ImageListAdapter;
+import com.team7.cmput301.android.theirisproject.IrisProjectApplication;
+import com.team7.cmput301.android.theirisproject.R;
+import com.team7.cmput301.android.theirisproject.controller.BodyPhotoListController;
+import com.team7.cmput301.android.theirisproject.controller.IrisController;
+import com.team7.cmput301.android.theirisproject.model.BodyPhoto;
+
+/**
+ * BodyPhotoListActivity is an activity to view the
+ * users body photos to be used in records
+ *
+ * @author itstc
+ * */
+public class BodyPhotoListActivity extends IrisActivity<BodyPhoto> {
+
+    private BodyPhotoListController controller;
+
+    private RecyclerView bodyPhotoList;
+    private FloatingActionButton addBodyPhotoButton;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_body_photos);
+
+        controller = createController(getIntent());
+
+        bodyPhotoList = findViewById(R.id.body_photo_list);
+        addBodyPhotoButton = findViewById(R.id.add_body_photo);
+
+        bodyPhotoList.setAdapter(new ImageListAdapter(this, controller.getBodyPhotos(), false));
+        bodyPhotoList.setLayoutManager(new GridLayoutManager(this, 3));
+
+        addBodyPhotoButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                dispatchAddBodyPhotoActivity(IrisProjectApplication.getCurrentUser().getId());
+            }
+        });
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+    }
+
+    private void dispatchAddBodyPhotoActivity(String id) {
+        Intent intent = new Intent(BodyPhotoListActivity.this, AddBodyPhotoActivity.class);
+        intent.putExtra("bodyphoto_user", id);
```

Comment:
Is BodyPhotoListActivity the only Activity that is going to use this intent extra? If so, refactor this to "com.team7.cmput301.android.theirisproject.bodyphoto_user" so it's more maintainable and doesn't have the possibility of conflicting with other apps' intents